app/models/profile.rb

```diff
@@ -14,7 +14,6 @@ class Profile < ApplicationRecord
     encryption_error: 2,
     verification_pending: 3,
     verification_cancelled: 4,
-    in_person_pending: 5,
```

Comment:
I think we should leave this just in case there is existing data out there, maybe rename it to `_deprecated` or something? But the risk of a collision when we do have a new event 5 make me hesitate

Reply:
> I think we should leave this just in case there is existing data out there, maybe rename it to `_deprecated` or something? But the risk of a collision when we do have a new event 5 make me hesitate

Yeah, we could. But since this was never enabled, maybe we could just confirm there's no instances of it in deployed environments, then be okay to just remove it altogether?

Reply:
Looks like `prod`, `staging` and `int` are clear:

```ruby
Profile.where(deactivation_reason: :in_person_pending).first
# nil
```

and to check that the syntax/encoding was correct, this returned results:
```ruby
Profile.where(deactivation_reason: :password_reset).first
```

Reply:
I checked `dev` as well, nothing there either.

I assume we're good to remove it outright?

Reply:
Yup, should be good to remove